doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -45,21 +46,25 @@ Similarly, an attempt to load CT schema with the already used version will resul
 }
 ```
 
-The CTL user interface automatically suggests the next available version (max loaded + 1) for a given FQN when creating a new CT schema.
-CT schema can have dependencies on other CT schemas.
-CT schema dependencies defined as an array of FQN-version combinations (identifiers of referenced CT schemas, see org.kaaproject.sample.ReferencedCT in the example above).
-Deleting CT schema is only permitted if its FQN-version combination is not referenced in any other CT schemas.
+Attempting to load a CT schema with no version or with a used version will result in an error.
+A CT schema can have dependencies on other CT schemas.
+CT schema dependencies are defined as an array of the CT schema IDs (for example, see org.kaaproject.sample.ReferencedCT schema in the code snippet above).
```

Comment:
This is type of "subsequent commenting on an example schema" I was referring to in line 21.